### ISSUE TITLE
[Windows] No implementation found on new window

### ISSUE THREAD

**Issue by CaoGiaHieu-dev** [pichillilorenzo/window_manager_plus#6]
when open new window , it doesnt registar the plugin . 
I have check example code and didnt see it too

**Comment by CaoGiaHieu-dev** [comment]
@pichillilorenzo Hi . can you take a look on this ? it should have some code like https://github.com/MixinNetwork/flutter-plugins/issues/58#issuecomment-1060371162

**Comment by pichillilorenzo** [comment]
Have you followed the setup correctly for Windows? Please check: https://github.com/pichillilorenzo/window_manager_plus?tab=readme-ov-file#windows
Doing it will result on plugins registration, if you didn’t change the basic generated FlutterWindow c++ file in a strange way.
A working example is my Flutter Browser App: https://github.com/pichillilorenzo/flutter_browser_app

**Comment by CaoGiaHieu-dev** [comment]
> Have you followed the setup correctly for Windows? Please check: https://github.com/pichillilorenzo/window_manager_plus?tab=readme-ov-file#windows Doing it will result on plugins registration, if you didn’t change the basic generated FlutterWindow c++ file in a strange way. A working example is my Flutter Browser App: https://github.com/pichillilorenzo/flutter_browser_app

yes . i was follow correctly as your document . and still when i open new window it crashed because No implementation found  window_manager_plus . 
i hasnt test Flutter Browser App . will check this later .
